src/pages/Game.js

```diff
@@ -29,12 +35,16 @@ export const Game = () => {
     <div className='App'>
       <header>
         <h1>Tic-Tac-Toe</h1>
-        <div className='user'>
-          <Button variant='contained' size='small'>
-            Change
-          </Button>
-        </div>
       </header>
+      <br />
+      <div className='finishedGame'>
```

Comment:
One of the requirements for the task was "the board has to be styled in a way, that allows the outcome message to be displayed, without changing the location of the board". Maybe it was not perfectly explained, but my idea was to not move the board at all when displaying the winner alert.